wrappers/core18.go

```diff
@@ -206,7 +206,7 @@ func AddSnapdSnapServices(s *snap.Info, opts *AddSnapdSnapServicesOptions, inter
 			Mode:    st.Mode(),
 		}
 	}
-	globs := []string{"snapd.service", "snapd.socket", "snapd.*.service", "snapd.*.timer"}
+	globs := []string{"snapd.service", "snapd.socket", "snapd.*.service", "snapd.*.timer", "snapd.*.target"}
```

Comment:
should this be covered in the unit tests here? also I wonder if we have any spread test about the details of what this does? I didn't see anything in particular, @mvo5 do your remember?

Reply:
The "add/change" case is implicitly tested of course when we installs snapd updates or seed snapd. But I don't think we test the remove cases here via spread. At least some unit tests for the "remove" case would be good.

Reply:
I can look next week into adding tests in a different PR.